src/BProject/BProject.Core/Model/Address.cs

```diff
@@ -0,0 +1,21 @@
+﻿using BProject.Core.Model.Base;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Bproject.Core.Model
+{
+    public class Address : BaseEntityWithTimestamp
+    {
+        public int? CustomerID { get; set; }
```

Comment:
CustomerID może być zwykłym int and nie int?, ponieważ deklarujemy, że jest required.